modules/oadp-verifying-upgrade-1-5-0.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="verifying-upgrade-1-5-0_{context}"]
+= Verifying the upgrade
+
+Use the following procedure to verify the upgrade.
```

Comment:
Same comment and suggestion as earlier
> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".
- https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc